src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests.vb

```diff
@@ -5243,7 +5243,37 @@ End Class
 
             comp.AssertTheseDiagnostics(
 <expected><![CDATA[
-DiagID1: 'C' is for evaluation purposes only and is subject to change or removal in future updates.
+BC30668: 'C' is obsolete: 'error'.
+    Sub M(c As C)
+               ~
+]]></expected>)
+        End Sub
+
+        <Fact>
+        Public Sub ExperimentalWithDiagnosticsId_WithObsolete_ReverseOrder()
```

Comment:
>ExperimentalWithDiagnosticsId_WithObsolete_ReverseOrder [](http://example.com/codeflow?start=19&length=55)

I think we should also test with Deprecated instead of Obsolete